R-packages/evalcast/R/scale_by_forecaster.R

```diff
@@ -47,17 +47,21 @@ scale_by_forecaster <- function(score_card,
     for (var in score_cols){
         base_values <- filter(score_card,
                               .data$forecaster == base_forecaster_name)[[var]]
-        if (!all(base_values > 0)) {
+        if (any(base_values <= 0)) {
```

Comment:
Shouldn't this just be `==`? I'm not sure what score metric would allow negative values, but that should be a separate warning if we want it.

Reply:
I think either would be fine. I switched `!all` to `any` because the first will fail if there's an `NA`. But it turns out that the second fails as well.